client/src/components/collectionPanel/CollectionSettingsTab/CollectionSettingsTab.tsx

```diff
@@ -0,0 +1,27 @@
+import GroupsInput from '../../groupsInput';
+import styles from './CollectionSettingsTab.module.css';
+
+type CollectionSettingsTabProps = {
+  groups: string[];
+  setGroups: (groups: string[]) => void;
+};
+
+export default function CollectionSettingsTab({
+  groups,
+  setGroups,
+}: CollectionSettingsTabProps) {
+  return (
+    <div>
+      <table className={styles.settingsTable}>
```

Comment:
Let's not use table tags for layouting. Rather use styled divs using flexbox or chakra components like Stack and HStack.